grpcurl.go

```diff
@@ -554,13 +554,12 @@ func invokeServerStream(ctx context.Context, stub grpcdynamic.Stub, md *desc.Met
 	return nil
 }
 
-func invokeBidi(ctx context.Context, cancel context.CancelFunc, stub grpcdynamic.Stub, md *desc.MethodDescriptor, handler InvocationEventHandler,
+func invokeBidi(ctx context.Context, stub grpcdynamic.Stub, md *desc.MethodDescriptor, handler InvocationEventHandler,
 	requestData RequestMessageSupplier, req proto.Message) error {
 
 	// invoke the RPC!
 	str, err := stub.InvokeRpcBidiStream(ctx, md)
 
-	// mutex protects access to handler and sendErr since we'll have two goroutines sharing them
```

Comment:
Also removed this errant comment, which does not apply (must have been describing earlier iteration of the code).